cnd/cnd.modelimpl/external/open-fortran-parser-0.7.1.2-notice.txt

```diff
@@ -0,0 +1,2 @@
+Open Fortran Parser v0.7.1.2
+Copyright (c) 2005, 2006 Los Alamos National Security, LLC.
```

Comment:
Why is this notice necessary? We include the full license, so the copyright is already present - am I missing something?

Reply:
You're right. I'll remove/squash/push during the weekend.